src/gui/newwizard/pages/accountconfiguredwizardpage.cpp

```diff
@@ -26,28 +26,18 @@ AccountConfiguredWizardPage::AccountConfiguredWizardPage(
     _ui->localDirectoryLineEdit->setText(QDir::toNativeSeparators(userChosenSyncTargetDir));
     _ui->syncEverythingRadioButton->setChecked(true);
 
-    // could also make it invisible, but then the UX is different for different installations
-    // this may be overwritten by a branding option (see below)
-    _ui->useVfsRadioButton->setEnabled(vfsIsAvailable);
+    _ui->useVfsRadioButton->setVisible(vfsIsAvailable);
 
     _ui->useVfsRadioButton->setText(tr("Use &virtual files instead of downloading content immediately"));
-    if (vfsModeIsExperimental) {
-        _ui->useVfsRadioButton->setIcon(Resources::getCoreIcon(QStringLiteral("warning")));
-
-        // when a feature is experimental and experimental features are disabled globally, it should be hidden
-        if (!Theme::instance()->enableExperimentalFeatures()) {
-            _ui->useVfsRadioButton->hide();
-        }
-    }
 
     // just adjusting the visibility should be sufficient for these branding options
     if (Theme::instance()->wizardSkipAdvancedPage()) {
         _ui->advancedConfigGroupBox->setVisible(false);
     }
 
-    if (!Theme::instance()->showVirtualFilesOption()) {
+    if (!Theme::instance()->forceVirtualFilesOption()) {
```

Comment:
Again, I don't see a reason for this change.

Reply:
Here we have a logic issue, I intended to hide if if vfs is enforced.
We really don't need to display it in that case.
(showVirtualFilesOption is removed)

Reply:
And I discovered that the whole section is hidden if vfs is forced